builder/dockerfile/instructions/parse_test.go

```diff
@@ -19,11 +18,11 @@ func TestCommandsExactlyOneArgument(t *testing.T) {
 		"STOPSIGNAL",
 	}
 
-	for _, command := range commands {
-		ast, err := parser.Parse(strings.NewReader(command))
+	for _, cmd := range commands {
```

Comment:
not directly related to this change, but my IDE kept highlighting this because it overlaps with the `github.com/docker/docker/builder/dockerfile/command` import 😅